src/convertToObject.js

```diff
@@ -3,16 +3,36 @@
 /**
  * Implement convertToObject function:
  *
- * Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
+ * Function takes a string with styles
+ * (see an example in [stylesString.js](./stylesString.js))
  * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
+ * and values are the values of related CSS properties
+ * (see an exampl in [test file](./convertToObject.test.js))
  *
  * @param {string} sourceString
  *
  * @return {object}
  */
 function convertToObject(sourceString) {
   // write your code here
+  // const stringToarray = sourceString.replace(/\s/g, '').split(';');
+
+  // const cssAtions = stringToarray.filter(action => action.split(':'));
+  // const cleaned = cssAtions.filter(action => action.length !== 0);
+  // const cssArray = cleaned.map(item => item.split(':'));
+
+  // const cssObject = cssArray.reduce((o, array) =>
+  //   Object.assign(o, { [array[0]]: array[1] }), {});
+
+  // return cssObject;
+
+  const cssObject = sourceString.replace(/\s\s+/g, ' ').split(';')
+    .filter(action => action.length > 1)
+    .map(item => item.split(':'))
+    .reduce((ob, array) =>
```

Comment:
Use array destructuring to make the code more concise and readable. And use more meaningful names for variables. What does `ob` mean?
```suggestion
    .reduce((ob, [property, value]) =>
```